index.html

```diff
@@ -151,1145 +151,1103 @@ <h4>Other Related Resources</h4>
         this document.</p>
       </section>
     </section>
-  </section>
 
 
-  <section id="h_arabic_script_overview">
-    <h2>Arabic Script Overview</h2>
+<section id="h_the_islamic_manuscript_tradition">
+        <h3>The Islamic manuscript tradition</h3>
 
+        <p>Islamic manuscript practice evolved with Islamic art and civilization. Extant evidence of decorative use of the Arabic script can be found as early as the 7th century CE, in a mosaic band in the Dome of the Rock. Beyond frequent monumental inscriptions in architecture, calligraphy became one of the foremost Islamic art forms. Manuscript practice evolved into a central element of Islamic civilization through the copying of books, and the refinement of calligraphy into a form of fine art. The commissioning and writing of the Qur‘an was seen as an expression of religious devotion and provided the context for scribes and calligraphers to excel in their craft. Further to its use as a textual medium, Arabic calligraphy and lettering crafts developed a rich palette of decorative forms and uses.</p>
+    <section id="writing_styles">
+        <h4>Writing Styles</h4>
+        <p>A range of styles evolved over time in different regions and for different purposes. These include monumental writing styles used in decorative and representational settings, styles used for the exclusive use of rulers, common hands in different areas, and styles dedicated for specialist purposes such as miniature writing. Many of these styles fell out of use, some evolved further, and some retained a high degree of consistency over time.</p>
+        <p>There is evidence of both, monumental and current hands to have existed and evolved from the early Islamic era onwards. Various exponents of the monumental styles are often loosely referred to as Kūfī, making it a term that lacks precision and clarity. It describes a group of styles that are marked by an angularity and stark graphic charcteristics that are absent from the round scripts that took preeminence from the late 10th, early 11th century CE.</p>
+        <p>The ‘Abbassīd calligrapher Ibn Muqlah (885/886–940 CE) is recognised for his contribution to the evolution of the round scripts. He articulated a system of proportional relations of Arabic letterforms, which became known as 'The Proportioned Script' (al-khatt al-mansūb), although its precise meaning remains open to interpretation. His work was continued by Ibn al-Bawwāb (d. 1022 CE), and later by Yāqūt al-Musta‘ṣimī (d. 1298 CE), who is credited with refining and canonising six writing styles into their definitive forms. The so-called 'Six Pens' (al-aqlām as-sitta) became the expression of an early high point of Islamic calligraphy. They are grouped in sister scripts of two, in which one is a large, the other a small hand, which share some characteristics. They are (large/small) Thuluth and Naskh, Muhaqqaq and Reyhān, and Tawqi‘ and Riqa‘, and they superceded all preceding styles.
```

Comment:
For the glossary we decided to use the LOC transcription.  I don't think we closed completely on what to do, however, for transcriptions of Arabic words in the main text. I see that your transcriptions aren't based on LOC. We probably have two choices: (a) use the LOC transcription (for consistency), or (b) use an Anglicised (ie. no diacritics) approximation.

Personally, i'd be fine with adopting LOC transcriptions.

Fwiw, @TitusNemeth we used https://r12a.github.io/pickers/arabic/ to generate such transcriptions consistently. (Add vowelled arabic to the box, then select Transcribe to LOC from the pulldown top right.) See also https://github.com/w3c/alreq/issues/204

Reply:
In fact, if we need to start a discussion on this topic before deciding what to do in the PR, let's do so at https://github.com/w3c/alreq/issues/204

Reply:
Understood and agreed, I thought I was following LOC but may have overlooked something.

Reply:
For رُقْعَة‎ the tool produces ruqʻah‎ (though it may be that it's acceptable without the h). 

The text uses Qur‘an for قُرآن, whereas the tool produces Qur’ān (glottal stop, rather than ain, and long a)

It's a while since i delved into the details of the algorithm to conver to LOC, but if you think the tool is incorrect, let me know and i'll look at fixing it. Cheers.

Reply:
Thanks, that's just my mistake